multiple-servers/webservers.tf

```diff
@@ -1,65 +1,66 @@
-resource "gridscale_server" "webserver1"{
-  name = "webserver1"
-  cores = 1
+resource "gridscale_server" "webserver1" {
+  name   = "webserver1"
+  cores  = 1
   memory = 1
   storage {
-    bootdevice = true
-    object_uuid = "${gridscale_storage.webserver1-storage.id}"
+    bootdevice  = true
+    object_uuid = gridscale_storage.webserver1-storage.id
   }
   network {
-    object_uuid = "${gridscale_network.webserver-network.id}"
+    object_uuid = gridscale_network.webserver-network.id
   }
 }
 
-resource "gridscale_storage" "webserver1-storage"{
-  name = "websever1-storage"
+resource "gridscale_storage" "webserver1-storage" {
```

Comment:
I would prefer not using the type of resource as a suffix in the name because we always refer to the resource with type.name.

Reply:
Your are right, same is true for network resources. WIll refactor this.